omod/src/main/java/org/openmrs/module/reporting/web/widget/handler/CohortIndicatorHandler.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
```

Comment:
Looks like you used an older license version.

Reply:
Sure, I actually just copied the license code at the top of the other indicator handler. What version of the license should it now be using?

Reply:
I created a jira, https://issues.openmrs.org/browse/REPORT-842 . I would gladly make that change also if you agree.

Reply:
You can copy the latest license from here https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/Auditable.java

Reply:
As for the JIRA project, that is very bright of you!!! 😊